lib/flutter_sound.dart

```diff
@@ -83,7 +83,7 @@ class FlutterSound {
     }
   }
 
-  Future<String> startRecorder(String uri, {double sampleRate = 44100.0, int numChannels = 2}) async {
+  Future<String> startRecorder(String uri, {int sampleRate = 44100, int numChannels = 2}) async {
```

Comment:
I thinks this should be `platform` specific value because in `ios`, it is receiving `double`. See the [line](https://github.com/dooboolab/flutter_sound/blob/388737aaa6451eaad98044a3a91481b29b92921c/ios/Classes/FlutterSoundPlugin.m#L147).